fuzz/client.c

```diff
@@ -65,13 +65,21 @@ int FuzzerTestOneInput(const uint8_t *buf, size_t len)
 
     /* This only fuzzes the initial flow from the client so far. */
     ctx = SSL_CTX_new(SSLv23_method());
+    if(ctx == NULL)
```

Comment:
Please always add space after `if`.

Reply:
Thanks a lot.
Should I resubmit?

Reply:
> Should I resubmit?

The best way is to fix the issues and commit with `git commit -a --fixup <commit-id-you're-fixing>`

This way the changes can be reviewed separately.

Reply:
OK,thanks